client/src/app/home/home/account-card/account-card.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { TransactionService } from '../transaction-card/services/transaction.service';
+import { Account } from '../../../models/account.model';
+import { AccountService } from './services/accounts.service';
+
+@Component({
+  selector: 'app-account-card',
+  templateUrl: './account-card.component.html',
+  styleUrls: ['./account-card.component.sass'],
+})
+export class AccountCardComponent implements OnInit {
+  @Input() public account!: Account;
+
+  public onCardClick(): void {
+    this.accountService.activateAccount(this.account._id);
+    this.transactionService.getTransactions(this.account._id).subscribe({
+      next: (data) => (this.transactionService.transactions = data),
```

Comment:
oh, i missed it.. It is bad to change value of service from component. If you need transactions here, create variable in this component

Reply:
oh yeah, this does nothing, I forgot to remove it.